.github/workflows/ci.yaml

```diff
@@ -0,0 +1,76 @@
+name: ci
+
+on: [push]
+
+env:
+  REGISTRY: justmark0
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          pip install pylint pytest
+          pip install --upgrade pip poetry==1.1.11 && poetry config virtualenvs.create false
+          cd app_python
+          poetry install
+      - name: Lint code
+        run: |
+          pylint $(git ls-files '*.py')
+        continue-on-error: true
+      - name: Test code
+        run: |
+          pytest
+  production:
+    needs: [ "build" ]
+    name: Push Image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
```

Comment:
Not sure that it does what you say

You are already passing `cache-from` and `cache-to` to the `build-push-action` below, it should be enough